runtime/lua/vim/diagnostic.lua

```diff
@@ -251,17 +251,26 @@ local function get_bufnr(bufnr)
   return bufnr
 end
 
----@private
-local function is_disabled(namespace, bufnr)
-  local ns = M.get_namespace(namespace)
-  if ns.disabled then
+--- Check diagnostic disabled in buffer
+---
+---@param bufnr number|nil Buffer number, or 0 for current buffer.
+---@param namespace number|nil Diagnostic namespace. When omitted, hide
+---                            diagnostics from all namespaces.
```

Comment:
```suggestion
---@param namespace number|nil Diagnostic namespace. When omitted, checks if
---                            all diagnostics are disabled in {bufnr}.
---                            Otherwise, only checks if diagnostics from
---                            {namespace} are disabled.
```